build: use MSVC name convention for atk,gtk,pango, cairo, gdk libs

boxerab opened the pull request:
This MR completes the work of previous MR #38 on glib library name format.

boxerab commented:
hey @thiblahute,  is this ready for merging ?

thiblahute commented:
What is missing here? Also could you please update the code with `ninja -C build update-code` ?

boxerab commented:
> What is missing here? Also could you please update the code with `ninja -C build update-code` ?

Thanks, I have updated the code with ninja and updated the PR. So, should all be ready for merge.
